src/RNGeth.ts

```diff
@@ -0,0 +1,127 @@
+import { NativeModules } from 'react-native'
+import { GethNativeModule } from "./GethNativeModule";
+
+export type NodeConfig = {
+  bootnodeEnodes?: string[],
+  enodes?: string,
+  genesis?: string,
+  ipcPath?: string
+  keyStoreDir?: string,
+  logFile?: string,
+  logFileLogLevel?: number,
+  maxPeers?: number,
+  networkID?: number,
+  noDiscovery?: boolean,
+  nodeDir?: string,
+  syncMode?: number
+  useLightweightKDF?: boolean,
+}
+
+export class RNGeth {
+  geth: GethNativeModule = NativeModules.RNGeth
+
+  /**
+   * Configure and prepare the node
+   * @returns success status of operation
+   */
+  async setConfig(config: NodeConfig) {
+    return this.geth.setConfig(config)
+  }
+
+  /**
+   * Start creates a live P2P node and starts running it.
+   * @returns success status of operation
+   */
+  async start(): Promise<boolean> {
+    return await this.geth.startNode()
```

Comment:
Nit: return await is redundant, can just return the promise
https://eslint.org/docs/rules/no-return-await